src/tasks/helpers/android.ts

```diff
@@ -1,8 +1,7 @@
 import { TaskExecutionContext, runningTasks } from '../tasksHelper';
 import { TaskHelper } from './base';
 import { CommandBuilder } from '../commandBuilder';
-import { selectAndroidKeystore } from '../../quickpicks/build/android';
-import { inputBox, enterPassword } from '../../quickpicks/common';
+import { enterAndroidKeystoreInfo } from '../../quickpicks/build/android';
 import { KeystoreInfo } from '../../types/common';
 import * as fs from 'fs-extra';
```

Comment:
<!--
  0 failure: 
  1 warning:  src/tasks/helpers...
  
  
  DangerID: danger-id-Danger;
  File: src/tasks/helpers/android.ts;
  Line: 6;
-->

- :warning: src/tasks/helpers/android.ts line 6 – 'fs' is defined but never used. (@typescript-eslint/no-unused-vars)